2009-02-08  Michael Tautschnig  <mt@debian.org>

	* lib/setup-storage/Sizes.pm: Properly tell the user that the lower bound of a
		parition size is already too high
Index: trunk/lib/setup-storage/Sizes.pm
===================================================================
--- trunk.orig/lib/setup-storage/Sizes.pm
+++ trunk/lib/setup-storage/Sizes.pm	
@@ -73,6 +73,8 @@
     $end = $end * 1024.0 * 1024.0;
   }
 
+  # the user may have specified a partition that is larger than the entire disk
+  ($start <= $size_b) or die "Lower bound of partition size is greater than disk size\n";
   # make sure that $end >= $start
   ($end >= $start) or &FAI::internal_error("end < start");
 
